src/child/actions/window.js

```diff
@@ -53,6 +58,59 @@ const updatingOptionsError = createActionCreator(() => ({
     type: ACTION_TYPES.UPDATING_OPTIONS_ERROR
 }));
 
+export const resizeWindow = createActionCreator((dimensions) => ({
+    type: ACTION_TYPES.RESIZE_WINDOW,
+    previousExpandedDimensions: dimensions
+}));
+
+export function resizeToMin() {
```

Comment:
Thanks for updating.

Just a follow up question - why are these dispatching `resize` actions internally? Same reason as before, I don't think they're actually resizing the window?